asciidoc/courses/aura-fundamentals/modules/3-services-tools/lessons/2-query/lesson.adoc

```diff
@@ -27,18 +27,23 @@ image::images/02_query_tool_start.jpg[querytool_start,width=500,align=center]
 
 The Query UI consists of the sidebar, the Cypher editor, and the result frame(s).
 
-As a good practice, you should remove the result-frames you don't want/need.
-
-Click on the three dots next to the frame title and select **Remove frame**.
 
    
+== Create query blocks using AI
+
+You can also create query blocks using AI by clicking the "Generate with AI" button. This will prompt you to enter a description of the data you want to query, and the AI will generate a Cypher query based on your description:
```

Comment:
There is a hanging "This".

What are "query blocks"? Should this just be queries?

```suggestion
You can generate queries from a prompt using the "Generate with AI" button. 

Enter a description of what you want to query and the Cypher will be created for you. For example, "FInd all movies".
```